src/main/java/net/mcreator/element/types/Dimension.java

```diff
@@ -93,6 +94,7 @@ public Dimension(ModElement element) {
 		// DEFAULT VALUES
 		this.enablePortal = true;
 		this.enableIgniter = true;
+		this.rarity = "UNCOMMON";

```

Comment:
This is not a valid default value and changes current default value

Reply:
You're right

Reply:
Done